markets/perps-market/subgraph/package.json

```
@@ -14,7 +14,7 @@
     "build:optimism-mainnet": "echo SKIP || node ./codegen optimism-mainnet && yarn codegen && graph build --output-dir ./build/optimism-mainnet --network optimism",
     "deploy:optimism-mainnet": "echo SKIP || graph deploy --output-dir ./build/optimism-mainnet --network optimism --product hosted-service snx-v3/perps-market-optimism-mainnet",
     "build:optimism-goerli": "node ./codegen optimism-goerli && yarn codegen && graph build --output-dir ./build/optimism-goerli --network optimism-goerli",
-    "deploy:optimism-goerli": "graph deploy --output-dir ./build/optimism-goerli --network optimism-goerli --product hosted-service snx-v3/perps-market-optimism-goerli",
+    "deploy:optimism-goerli": "graph deploy --output-dir ./build/optimism-goerli --network optimism-goerli --product hosted-service rickk137/v3-perps-opt-goerli",
```

```suggestion
    "deploy:optimism-goerli": "graph deploy --output-dir ./build/optimism-goerli --network optimism-goerli --product hosted-service snx-v3/v3-perps-opt-goerli",
```
Looks like you left your personal endpoint